contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -774,7 +776,12 @@ contract BosonVoucherBase is IBosonVoucher, BeaconClientBase, OwnableUpgradeable
                 IBosonFundsHandler(protocolDiamond).depositFunds{ value: balance }(sellerId, token, balance);
```

Comment:
```suggestion
                IBosonFundsHandler(protocolDiamond).depositFunds{ value: balance }(sellerId, token, balance);
```
Not directly connected to this PR, but I guess we could add this to prevent unnecessary calls.